routes/messages.js

```diff
@@ -55,20 +53,20 @@ module.exports = (db) => {
   });
 
 
-  // add new message
+  // add new message from item_details
   router.post("/", (req, res) => {
     let queryString = `
     INSERT INTO messages(sender_id, receiver_id, item_id, body)
     VALUES($1, $2, $3, $4)
     RETURNING *;
     `
-    let queryParams = [req.session.user_id, req.body.receiver_id, req.body.item_id, req.body.message]
     console.log(req.body);
+    let queryParams = [req.session.user_id, req.body.receiver_id, req.body.item_id, req.body.message];
+
     db.query(queryString,queryParams)
     .then(data => {
       const messages = data.rows;
-      console.log(messages);
-      res.redirect('/api/messages');
+      res.redirect("/");
```

Comment:
Why not redirect them to their messages?

Reply:
Like /api/messages/:id ? or messages?

Reply:
If you redirect /api/message it will not show the message you sent, so for the demo purpose it may not look great i thought and I am not sure how I can redirect to /api/message/:id ...